### PR TITLE
Add subdomains to sn06's certbot for wildcard certificate

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**

### CODE REVIEW

File: sn06.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@
       - "*.interactivetoolentrypoint.interactivetool.usegalaxy.eu"
       - "*.interactivetoolentrypoint.interactivetool.live.usegalaxy.eu"
       - "*.interactivetoolentrypoint.interactivetool.test.usegalaxy.eu"
+      - "*.aquainfra.usegalaxy.eu"
```
Comment (bgruening) [inline sn06.yml:42]:
```suggestion
      - "*.aquainfra.usegalaxy.eu"
      - "*.ecology.usegalaxy.eu"
```

That will also make @yvanlebras happy.

Comment (bgruening) [reply sn06.yml:42]: Oh, they named themself `aqua`.

Comment (yvanlebras) [reply sn06.yml:42]: Is it what I think ? ;)

**Review by bgruening — APPROVED** [review]

**Comment by sanjaysrikakulam** [comment]
Shall I attempt the wildcard cert creation?

**Comment by bgruening** [comment]
As you like! :) YOLO!

**Comment by sanjaysrikakulam** [comment]
> Do we need to add this to our operations? In particular to https://usegalaxy-eu.github.io/operations/subdomains.html

Sure, I will add a sub-section there on how to for GxITs.

**Comment by sanjaysrikakulam** [comment]
Now that the certificates are in place, I am merging this PR.

**Pull request merged**